modules/Kconfig.cmsis

```diff
@@ -19,3 +19,12 @@ config HAS_CMSIS_CORE_M
 	bool
 
 endif
+
+menuconfig CMSIS_DSP
+	bool "CMSIS-DSP Library Support"
+	depends on CPU_CORTEX
+	depends on NEWLIB_LIBC
```

Comment:
why do we need NEWLIB for the DSP lib?

Reply:
CMSIS-DSP makes use of the standard C math functions that not available in the Zephyr minimal libc (e.g. `sqrtf`, `logf`, `expf`, `fabsf`).

If you are doing DSP work, you will have the full C library (i.e. newlib) enabled anyway, so it should be too much of a problem.

Reply:
That's a very good question, I wondered the same myself but forgot to ask.

Reply:
@stephanosio ^^

Reply:
@carlescufi Did you see the answer above? (https://github.com/zephyrproject-rtos/zephyr/pull/24307#discussion_r412961171)